src/catch2/internal/catch_template_test_registry.hpp

```diff
@@ -84,11 +84,12 @@
             struct TestName{\
                 TestName(){\
                     size_t index = 0;                                    \
-                    constexpr char const* tmpl_types[] = {CATCH_REC_LIST(INTERNAL_CATCH_STRINGIZE_WITHOUT_PARENS, __VA_ARGS__)};\
-                    using expander = size_t[];\
+                    constexpr char const* tmpl_types[] = {CATCH_REC_LIST(INTERNAL_CATCH_STRINGIZE_WITHOUT_PARENS, __VA_ARGS__)}; /* NOLINT(cppcoreguidelines-avoid-c-arrays,modernize-avoid-c-arrays) */\
```

Comment:
This warning has at least one more alias: `hicpp-avoid-c-arrays`
```suggestion
                    constexpr char const* tmpl_types[] = {CATCH_REC_LIST(INTERNAL_CATCH_STRINGIZE_WITHOUT_PARENS, __VA_ARGS__)}; /* NOLINT(cppcoreguidelines-avoid-c-arrays,modernize-avoid-c-arrays,hicpp-avoid-c-arrays) */\
```

([clang-tidy's handling of aliases is pretty damn bad](https://github.com/llvm/llvm-project/issues/54319))

Reply:
Thanks for the catch. I have added the alias now. I reviewed the clang-tidy page and it seems adding `hicpp-avoid-c-arrays` should cover all the aliases: https://clang.llvm.org/extra/clang-tidy/checks/modernize/avoid-c-arrays.html

Reply:
Added the requested alias: b6e4e839